### PR TITLE
Completed css-two-column-layout exercise

### PR DESCRIPTION
**Pull request by wang-morris**
Screen Shot 2022-05-25 at 9 43 49 PM

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      </div>
+    </div>
+    <div class="row">
+      <div class="images">
+        <img src="images/software.svg" alt="image of software that looks like Africa">
+      </div>
+      <div class="two-fifths">
+        <h3>INTUITIVE,<br>SMART SOFTWARE</h3>
+        <p>We build products that transform how healthcare facilties run across Africa &#8212; making them radically more efficient operationally and financially, a complete turnaround.</p>
+      </div>
```
Comment (BrettAlbright) [inline css-two-column-layout/index.html]: the only thing to add would be to wrap these elements with a `column-half` element which will be the empty box for half of the row.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
looking good! one small update!

**Comment by wang-morris** [comment]
Screen Shot 2022-05-26 at 1 46 58 PM

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**